contribs/gmf/src/directives/mouseposition.js

```diff
@@ -64,32 +65,64 @@ gmf.MousepositionController = function($element, $filter, gettextCatalog) {
    */
   this.projection;
 
+  /**
+   * @type {angular.Scope}
+   *
+   */
+  this.$scope_ = $scope;
+
+  /**
+   * @type {angularGettext.Catalog}
+   * @private
+   */
+  this.gettextCatalog_ = gettextCatalog;
```

Comment:
FWIW, you can mark it protected and remove the trailing underscore. Then there is no need to alias it in the code below.